src/main/resources/com/google/api/codegen/php/method_sample.snip

```diff
@@ -1,14 +1,30 @@
 @extends "php/common.snip"
 
-@snippet decorateSampleCode(apiMethod, coreSampleCode)
+@snippet decorateSampleCode(xapiClass, apiMethod, coreSampleCode)
     try {
-        ${@apiMethod.apiVariableName} = new {@apiMethod.apiClassName}();
+        @if xapiClass.hasMissingDefaultOptions
+            $options = [
+                @if xapiClass.missingDefaultServiceAddress
+                    'serviceAddress' => 'my-service-address',
+                @end
+                @if xapiClass.missingDefaultServiceScopes
+                    'scopes' => ['my-service-scope'],
```

Comment:
Why not make my-service-scope and my-service-address defaults if you still have to set them here?